packages/internal-utils/src/poaBridge/getPendingDeposits.ts

```
@@ -2,23 +2,37 @@ import type { IntentsUserId } from "../types/intentsUserId";
+	do {
+		const result = await getDepositStatus({
+			account_id: accountId,
+			limit,
+			offset,
+		});
+		result.deposits.forEach((deposit) => {
+			if (isPending(deposit)) pendingDeposits.push(deposit);
+		});
+		offset += result.deposits.length;
+		if (result.deposits.length < limit) break;
+	} while (true);
```

Biome (2.4.9)

[error] Unexpected constant condition.

(lint/correctness/noConstantCondition)